dokka-integration-tests/gradle/projects/it-android/settings.gradle.kts

```diff
@@ -0,0 +1,64 @@
+rootProject.name = "it-android-0"
+
+pluginManagement {
```

Comment:
Yes, the config is duplicated. Since these tests are a fresh start, I wanted to try and avoid using the existing test utilities initially, to try and see that would encourage some alternatives. 

Copy and pasting the config isn't ideal, but it works for now. I would like to revisit it in the future when more tests are added, and it becomes a bit more clear about how the DGPv2 integration tests can be best implemented.

Is it okay to leave the copy-pasted code for now?

Reply:
yeah, okay, let's keep it like this for now